app/Repositories/WordRepository.php

```diff
@@ -2,70 +2,86 @@
 
 namespace App\Repositories;
 
+use App\Language;
 use App\Word;
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Http\Request;
 
-class WordRepository implements RespositoryInterface
+class WordRepository
 {
-    protected $word;
-
-    public function __construct(Word $word)
-    {
-        $this->word = $word;
-    }
-
     public function all()
     {
-        return $this->word->all();
+        return Word::all();
     }
 
     /**
      * Get all requested words.
      *
-     * @return Word[]|\Illuminate\Database\Eloquent\Builder[]|\Illuminate\Database\Eloquent\Collection
+     * @return Word[]|\Illuminate\Database\Eloquent\Builder[]|Collection
      */
-    public function allRequested()
+    public function allRequested(): Word
     {
-        return $this->word->has('requesters')->get();
+        return Word::has('requesters')->get();
     }
 
     /**
      * Get all words with at least one sign.
      *
-     * @return Word[]|\Illuminate\Database\Eloquent\Builder[]|\Illuminate\Database\Eloquent\Collection
+     * @return Word[]|\Illuminate\Database\Eloquent\Builder[]|Collection
      */
-    public function allSigned()
+    public function allSigned(): Word
     {
-        return $this->word->has('signs')->get();
+        return Word::has('signs')->get();
     }
 
-    public function allVacant()
+    public function allVacant(): Word
```

Comment:
Flest her virker ikke med `: Word` return statement, især da med `->get()` returnerer du en Collection og ikke én model

Reply:
Ændret til collection